schemas/2016-02-03/Microsoft.Devices.json

```diff
@@ -355,6 +370,32 @@
         }
       },
       "description": "The properties of the Storage Endpoint provided by the customer for file upload"
+    },
+    "IpFilterRule": {
+      "description": "IP filter Rule",
+      "type": "object",
+      "properties": {
+        "filterName": {
```

Comment:
To support customer to use template expression for any properties, we need to have following reference: "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"